english/selfie/selfie.md

```diff
@@ -8,10 +8,16 @@ Oh no! You are made of marshmallow!
 
 This sudden revelation has now complicated the matter.
 
-You hate marshmallows but can you really hate yourself?  
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.